charts/cronjob/README.md

```diff
@@ -57,6 +57,7 @@ The chart can be customized using the following configurable parameters:
 | `image.args` | docker Entrypoint command arguments array. The docker image's `CMD` is used if this is not provided. | `[]` |
 | `resources` | resource requests and limits | `{}` |
 | `schedule` | schedule of the cronjob | `*/1 * * * *` |
+| `startingDeadlineSeconds` | By default if job misses 100 times it's scheduled successful execution (due to jobs's unexpectedly long run or temporal network error) then cronjob gets stuck and do not schedule new jobs anymore. This setting ensures that failed job executions are only counted within last 600 seconds and thus not reaching the limit. For cron schedules below 1 minutes one should consider to override default with lower value. More info: https://github.com/kubernetes/kubernetes/pull/39608 | 600 |
```

Comment:
if job misses 100 times it's scheduled successful execution -> if job is not successfully finished after 100 times of repeating

Reply:
https://github.com/commercetools/k8s-charts/pull/18/commits/23bf8dec138584798d5a0be579c0a5728f38418a